scip-java/src/main/scala/com/sourcegraph/scip_java/buildtools/ScipBuildTool.scala

```diff
@@ -773,6 +780,9 @@ class ScipBuildTool(index: IndexCommand) extends BuildTool("SCIP", index) {
       reportWarningOnEmptyIndex: Boolean = true,
       javaHome: Option[String] = None,
       dependencies: List[Dependency] = Nil,
+      // If repositories aren't provided at all, use all the defaults
+      // if repositories are set, we only use the ones provided explicitly
+      repositories: Option[List[String]] = None,
```

Comment:
Is the expectation that users manually write this? My thinking so far has been to rely on `COURSIER_REPOSITORIES` (https://get-coursier.io/docs/other-repositories) and `COURSIER_CREDENTIALS`, which can be overridden from executors without modifying the scip-java.json file. I haven't validated this works, however.